recipes/default.rb

```diff
@@ -34,4 +34,4 @@
   node.default['hive']['hive_env']['java_home'] = node['java']['java_home']
 end
 
-include_recipe 'hadoop::default'
```

Comment:
Yes. We shouldn't have been forcibly running the `hadoop` cookbook. I was only doing it to get the user accounts, which we now simply create ourselves. Instead, the user should specify the recipes they want to run/wrap.